scripts/deepops/ansible-lint-roles.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
Needs a line or 2 of comments documenting the usage of the script.

Reply:
Done, see below.